frontend/lang/french.json

```diff
@@ -0,0 +1,337 @@
+    "t_key_selection_description": "Sélectionne plusieurs cellules en traçant une zone de sélection depuis l'espace entre les cellules.",
+    
+    "t_key_or": "où",
+    
+    "t_navigate_to_previous_page": "Retour",
+    
+    "t_bonds_static_warning": "Tu regardes un document statique. {{run_notebook_action}} pour activer l'interactivité.",
+    "t_bonds_static_warning_inner": "Exécuter ce notebook",
+    
+    "t_edit_or_run_this_notebook": "<strong>Éditer</strong> ou <strong>exécuter</strong> ce notebook",
+    "t_edit_or_run_view_code": "Voir le code",
+    "t_edit_or_run_view_code_cancel": "Annuler",
+    "t_edit_or_run_description_1": "Pour pouvoir éditer le code et exécuter les cellules, tu dois exécuter le notebook toi-même.",
+    "t_edit_or_run_description_2": "Où veux-tu exécuter le notebook ?",
+    "t_binder_help_text_title": "Dans le cloud <em>(expérimental)</em>",
+    "t_binder_help_text": "<a target='_blank' href='https://mybinder.org/'>Binder</a> est un service gratuit et open-source qui exécute des notebooks scientifiques dans le cloud ! Cela prendra du temps, généralement 2-7 minutes, pour obtenir une session.",
+    "t_edit_or_run_runtime": "Ce notebook prend environ <span>{{runtime}}</span> pour s'exécuter.",
+    "t_edit_or_run_local": "Sur ton ordinateur",
+    "t_edit_or_run_local_description": "(Recommandé si tu veux stocker tes changements.)",
+    "t_edit_or_run_download_notebook": "Télécharger le notebook :",
+    "t_edit_or_run_copy_notebook_url": "Copier l'URL du notebook :",
+    "t_edit_or_run_run_pluto": "Exécuter Pluto",
+    "t_edit_or_run_run_pluto_description": "Voir aussi : <a target='_blank' href='{{url}}'>Comment installer Julia et Pluto</a>",
+    "t_edit_or_run_open_the_notebook_file": "Ouvrir le fichier notebook",
+    "t_edit_or_run_open_the_notebook_file_description": "Taper le nom du fichier sauvegardé dans la boîte <em>ouvrir</em>.",
+    "t_edit_or_run_paste_url_in_the_open_box": "Coller l'URL dans la boîte <em>Ouvrir</em>",
+    
+    "t_time_minutes_one": "{{count}} minute",
+    "t_time_minutes_other": "{{count}} minutes",
+    "t_time_seconds_one": "{{count}} seconde",
+    "t_time_seconds_other": "{{count}} secondes",
+    
+    "t_frontmatter_title": "Métadonnées",
+    "t_frontmatter_description": "Si tu publies ce notebook sur le web, tu peux définir les paramètres ci-dessous pour fournir des métadonnées HTML. C'est utile pour les moteurs de recherche et les réseaux sociaux.",
+    "t_frontmatter_synchronized": "Métadonnées synchronisées ✔\n\nCes paramètres seront utilisés dans les futures exportations.",
+    "t_frontmatter_delete_field": "Supprimer le champ",
+    "t_frontmatter_add_field": "Ajouter un champ {{plus}}",
+    "t_frontmatter_add_author": "Ajouter un auteur {{plus}}",
+    "t_frontmatter_cancel": "Annuler",
+    "t_frontmatter_save": "Sauvegarder",
+    "t_frontmatter_preview": "Aperçu",
+    
```

Comment:
Do we want `t_frontmatter_language_placeholder` for French too? It seems to have been forgotten

Reply:
https://github.com/fonsp/Pluto.jl/blob/ecc3294659b32450bf3d772c225b9e1c8bcdf658/frontend/lang/english.json#L268

Reply:
oh right, I have not rebased on main 🤦